static/src/javascripts/projects/commercial/modules/comscore.spec.js

```diff
@@ -30,17 +25,14 @@ describe('comscore init', () => {
         commercialFeatures.comscore = false;
         init();
 
-        expect(onGuConsentNotification).not.toBeCalled();
+        expect(onConsentChange).not.toBeCalled();
     });
 
-    it('should register a callback with onGuConsentNotification if enabled in commercial features', () => {
+    it('should register a callback with onConsentChange if enabled in commercial features', () => {
         commercialFeatures.comscore = true;
         init();
 
-        expect(onGuConsentNotification).toBeCalledWith(
-            'performance',
-            _.initOnConsent
-        );
+        expect(onConsentChange).toBeCalled();
```

Comment:
Maybe checking the parameter still makes sense?
```suggestion
        expect(onConsentChange).toBeCalledWith(_.initOnConsent);
```